runtime/runc_jailer.go

```diff
@@ -447,6 +440,13 @@ func (j *runcJailer) bindMountFileToJail(src, dst string) error {
 	}
 	defer f.Close()
 
+	if strings.Contains(dst, "fifo") {
```

Comment:
I do think this is too hacky. I know we need to chown the fifo files. Are there files that we cannot chown?

Reply:
I also don't like the hardcoding, fixed it.